Create prioritization_response.md (#37)
octokit/.github: community/prioritization_response.md

Add that to the number of repos that we need to maintain (70+) and weekly interactions (500-1000 +/-) that we end up having with everyone I am sure it's clear why we have to define how we **prioritize** and our commitment to respond (think of an **SLA** of sorts) to those priorities.

### What are the Priorities?

In each of the repositories we own you'll see the same [labels](https://github.com/octokit/octokit.net/labels) for priority.  

**They are:**  
`Priority: Critical`, `Priority: High`, `Priority: Normal`, `Priority: Low`

Incidentally, these map directly to the priorities set in our [Octokit Active community board](https://github.com/orgs/octokit/projects/10/views/4).

| [Repo label](https://github.com/octokit/octokit.net/labels)   |      [Community board](https://github.com/orgs/octokit/projects/10/views/4) | Example |
|----------|:-------------:|:-----------|
| Priority: Critical | Urgent | Any security issues - Data leaks - Broken or non functioning SDK that has been published - Any stop work scenarios for users (where rollback is not possible)|

> | Priority: High | High | Any broken SDK methods - Deprecated or broken features - Critical dependency updates - Pull Requests that have been completed, are active, and are passing CI - Any issues that have been reported multiple times / impacting multiple users |

Reviewer: Are these dashes supposed to be a bulleted list or not? They're not [rendering](https://github.com/octokit/.github/blob/f68e4bf2f6431a45099a110cf289b40a270a84b9/community/prioritization_response.md) as one right now.

Author: They were just meant to be separators.  If you come up with a better layout let me know!

Reviewer: That's a bit of a tough one. I'm :+1: to leave them as they are right now; I don't have any better ideas.

| Priority: Normal | Normal | Any non-prioritized features - Issues impacting a low number of users - Schema changes and additions - Normal dependency updates |
| Priority: Low | Low | Basic workflow changes - Draft issues/PRs - Stale issues/PRs - non-critical dependency updates - Any incomplete design work |

### What are our response commitments?

Note: This does not represent a commitment to resolution, just how quickly we'll respond and begin working on it

| Priority  | Response expectation | 
|----------|:-----------|

> | Urgent | 0 to **24hrs**:  Communication with users is ongoing and frequent until the issue is resolved. A security advisory will be posted if needed. |

Reviewer: I'm a little nervous about committing to this. Does this mean an update at least every 24 hours on each urgent issue? Our current schedule right now doesn't budget for that, for example.

Author: Agreed, this was more a 👍 in the air that I was hoping to use for discussion.

Reviewer: I like the idea definitely! I think my worry was introducing an expectation that we would provide daily support or updates until the issue is resolved, which I don't think is reflective of reality at this point unfortunately.

> | Urgent | 0 to **24hrs**:  Communication with users is ongoing and frequent until the issue is resolved. A security advisory will be posted if needed. |

Author: @kfcampbell how does this sound? 

```suggestion
| Urgent | 0 to **24hrs**:  A security advisory will be posted if needed and updates publically posted to the corresponding repository |
```

Reviewer: Looks good to me!

| High | =< **1 Week** : Typically, we'll respond in issue/pr and will have a plan of action within the response expectation time window. | 
| Normal | >= **1 Week** : We'll acknowledge and make administrative assignments to the issue or PR.  They might get labeled "up-for-grabs". |
| Low | > **1 Week**: Typically, little to no action will take place on these items. They will often get labeled "up-for-grabs". |

### How do we Prioritize?

Remembering that we as a team have to prioritize not at the individual SDK or community level but across many repositories and communities our, methods need to consider several variables.

> Our process, while not exactly scientific, leans heavily into a broad stroke that favors protecting :lock: and serving :hotel: our communities first - we use a modified version of [RICE](https://www.productplan.com/glossary/rice-scoring-model/).

Reviewer: Is the police motto language here intentional?

Author: Haha that's funny.  As I was writing it, I was like "this phrase is so familiar, where have I heard that".  We can nix it for sure. 😜

Reviewer: Perhaps something along the lines of "Our process, while not exactly scientific, is aimed at protecting the interests of our communities first - we use a modified version of [RICE](https://www.productplan.com/glossary/rice-scoring-model/)."

Thoughts?

Author: Yeah I like that 👍

> Our process, while not exactly scientific, leans heavily into a broad stroke that favors protecting :lock: and serving :hotel: our communities first - we use a modified version of [RICE](https://www.productplan.com/glossary/rice-scoring-model/).

Author: ```suggestion
Our process, while not exactly scientific, is aimed at protecting the interests of our communities first - we use a modified version of [RICE](https://www.productplan.com/glossary/rice-scoring-model/).
```
